src/Server/Templates.fs

```diff
@@ -0,0 +1,41 @@
+module ServerCode.Templates
+
+
+
+open System.Threading.Tasks
+open Microsoft.AspNetCore.Http
+open Giraffe
+open ServerCode.Domain
+open ServerCode.FableJson
+open ServerTypes
+open Client.Shared
+open Giraffe.GiraffeViewEngine
+open Fable.Helpers.ReactServer
+
+let index (model: Model) =
+  let jsonState = toJson (toJson model)
+  let htmlStr = Client.Shared.view model ignore |> renderToString
+  html []
+    [ head [] [ meta [ _httpEquiv "Content-Type"; _content "text/html"; _charset "utf-8" ] ]
+      title [] [ rawText "SAFE-Stack sample" ]
+      link
+        [ _rel "stylesheet"
+          _href "https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css";
+          attr "integrity" "sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
```

Comment:
what's this thing?

Reply:
It's a html5 attribute to verify the subresource, see: https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity

The `Template.fs` is simply transformed from the `index.html` file: https://github.com/SAFE-Stack/SAFE-BookStore/blob/master/src/Client/index.html#L6